### PR TITLE
Update unifi-poller template to v2

### PR DESCRIPTION
**Pull request by davidnewhall**
This updates the poller template to v2. You may merge this any time. v2 isn't released yet, but will be very soon!

### CODE REVIEW

**Comment by davidnewhall** [comment]
ugh not sure why it's conflicting. :(

File: templates/unifi-poller.xml [hunk 1]
```diff
@@ -3,25 +3,23 @@
...
-  <Project>https://github.com/davidnewhall/unifi-poller</Project>
-  <Overview>Collect ALL UniFi Controller, Device and Client Data - Export to InfluxDB or Prometheus. Visualize with Grafana using included dashboards!&#xD;
-&#xD;
-This template works with poller versions 1.6.2+&#xD;
-&#xD;
-READ THE INSTRUCTIONS! &#xD;
-https://github.com/davidnewhall/unifi-poller/wiki/Installation</Overview>
-  <Category>Tools:Utilities Network:Management </Category>
+  <Changes>See https://github.com/unifi-poller/unifi-poller/wiki/CHANGES</Changes>
+  <Support>https://community.ui.com/questions/UniFi-Poller-Store-UniFi-Controller-Metrics-in-Prometheus-or-InfluxDB/58a0ea34-d2b3-41cd-93bb-d95d3896d1a1</Support>
```
Comment (Roxedus) [inline templates/unifi-poller.xml]: Did you read my "Do not do this"? :stuck_out_tongue:

File: templates/unifi-poller.xml [hunk 2]
```diff
@@ -3,25 +3,23 @@
...
-&#xD;
-This template works with poller versions 1.6.2+&#xD;
-&#xD;
-READ THE INSTRUCTIONS! &#xD;
-https://github.com/davidnewhall/unifi-poller/wiki/Installation</Description>
+  <Icon>https://raw.githubusercontent.com/wiki/unifi-poller/unifi-poller/images/unifi-poller-logo.png</Icon>
+  <Overview>
+    Collect ALL UniFi Controller, Device and Client Data - Export to InfluxDB or Prometheus.
+    Visualize with Grafana using included dashboards!
+    IMPORTAT! ACTION REQUIRED! As of UniFi Poller version 2 all of the environment variables and config file format changed.
```
Comment (Roxedus) [inline templates/unifi-poller.xml]: Can be highlighted
ref: https://github.com/selfhosters/unRAID-CA-templates/blob/master/templates/monitorr.xml#L10-L11

File: templates/unifi-poller.xml [hunk 3]
```diff
@@ -31,110 +29,105 @@ https://github.com/davidnewhall/unifi-poller/wiki/Installation</Description>
...
-      <Name>UP_QUIET</Name>
-      <Mode/>
+      <Value></Value>
+      <Name>UP_PROMETHEUS_DISABLE</Name>
+    </Variable>
+    <Variable>
+      <Value></Value>
+      <Name>UP_PROMETHEUS_HTTP_LISTEN</Name>
     </Variable>
   </Environment>
```
Comment (Roxedus) [inline templates/unifi-poller.xml]: You can [shave off the xml](https://selfhosters.net/docker/templating/templating/#114-shave-off-the-xml)

File: templates/unifi-poller.xml [hunk 4]
```diff
@@ -31,110 +29,105 @@ https://github.com/davidnewhall/unifi-poller/wiki/Installation</Description>
...
-  <Config Name="UniFi Re-authenticate" Target="UP_REAUTHENTICATE" Default="false" Mode="" Description="Container Variable: UP_REAUTHENTICATE&#13;&#10;- This is a debug method. Do not enable unless instructed to do so." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="UniFi Verify SSL" Target="UP_VERIFY_SSL" Default="false" Mode="" Description="Container Variable: UP_VERIFY_SSL&#13;&#10;- Enable this if you have a valid SSL cert on your controller." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="Quiet mode" Target="UP_QUIET" Default="false" Mode="" Description="Container Variable: UP_QUIET&#13;&#10;- Disables most log output." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
+  <Labels/>
+  <!-- poller -->
+  <Config Name="Poller Debug Mode" Target="UP_POLLER_DEBUG" Default="true|false" Mode="" Description="Container Variable: UP_POLLER_DEBUG - Setting true includes more log output." Type="Variable" Display="always-hide" Required="false" Mask="false">false</Config>
```
Comment (Roxedus) [inline templates/unifi-poller.xml]: The bool wont work, because a value is already set.

File: templates/unifi-poller.xml (hunk 3 above)
Comment (davidnewhall) [reply templates/unifi-poller.xml]: oh shoot, those aren't actually needed? That sure makes it nicer to look at.

File: templates/unifi-poller.xml (hunk 4 above)
Comment (davidnewhall) [reply templates/unifi-poller.xml]: Does it fail silently? It seemed to work..... I'll check again!

File: templates/unifi-poller.xml (hunk 2 above)
Comment (davidnewhall) [reply templates/unifi-poller.xml]: That's a great idea, let me play with that!

File: templates/unifi-poller.xml (hunk 1 above)
Comment (davidnewhall) [reply templates/unifi-poller.xml]: I did not lol

File: templates/unifi-poller.xml (hunk 4 above)
Comment (Roxedus) [reply templates/unifi-poller.xml]: in reality you are just choosing for the user. guess thats fine

Comment (davidnewhall) [reply templates/unifi-poller.xml]: I played with this. If you add `true` or `false` between `<Config>` and `</Config>` it takes that as the default value. You can still change it. If you leave the value missing from between the open and close tag it takes the first value inside the `Default` param. For example you can set a default of `true` by passing `Default="true|false"` and you can set a default of `false` with `Default="false|true"`. I went ahead and did it this second way.

**Comment by davidnewhall** [comment]
Thanks for the feedback! I used the highlighting and html example you provided. It looks a little nicer now. I also trimmed much of the unneeded xml. I did not realize `environment` and `config` were superfluous, so thank you again!

**Pull request merged**